src/index.js

```diff
@@ -1,11 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
It would be best if you left `index.js` untouched. Use `App.js` as your root component. If you intend to keep the code in `App`  minimal you can add the routes to its own module.